### PR TITLE
Perl syntax highlighting

### PR DESCRIPTION
**Pull request by MegOBonus**
Add syntax highlighting for Perl

### CODE REVIEW

File: build/templates/onedark.template.vim [hunk 1]
```diff
@@ -421,6 +421,21 @@ call s:h("xmlEndTag", { "fg": s:red })
...
+call s:h("perlFunction", { "fg": s:purple })
+call s:h("perlMethod", { "fg": s:purple })
+call s:h("perlStatementFiledesc", { "fg": s:red })
+call s:h("perlVarPlain",{ "fg": s:blue })
+call s:h("perlSharpBang", { "fg": s:comment_grey })
+call s:h("perlStatementInclude", { "fg": s:purple })
+call s:h("perlStatementScalar",{ "fg": s:purple })
+call s:h("perlSubName",{ "fg": s:yellow })
+call s:h("perlSpecialString",{ "fg": s:cyan })
+
```
Comment (simonsmith) [inline build/templates/onedark.template.vim]: You just need to [run the build step](https://github.com/joshdick/onedark.vim/blob/master/CONTRIBUTING.md#use-it) to create the real files from the templates and then this is good to go!

Comment (MegOBonus) [inline build/templates/onedark.template.vim]: Ok)

Comment (MegOBonus) [inline build/templates/onedark.template.vim]: Hmm.I have some troubles with a build. Can you build it by yourself?

Comment (simonsmith) [inline build/templates/onedark.template.vim]: Sure I can sort this.

Comment (MegOBonus) [inline build/templates/onedark.template.vim]: Thanks

**Pull request closed without merging**